### ISSUE TITLE
Loss function of BreastPathQ fine-tuning

### ISSUE THREAD

**Issue by bguetarni** [srinidhiPY/SSL_CR_Histo#9]
Hi, thank you for sharing your code and pre-trained models !

I have a question regarding the loss function used for fine-tuning the pre-trained model on the BreastPathQ dataset.
In [line 387](https://github.com/srinidhiPY/SSL_CR_Histo/blob/adc301ec74b69defc51cfe2ab3effccc7c1f60c5/eval_BreastPathQ_SSL.py#L387) it looks like you use the mean squarred error, although you build your model as a classifier. Is this what you wanted to do ? And if yes, could you explain why ?

Thank you.

**Comment by bguetarni** [comment]
I apologize, I realized that this dataset is effectively doing regression.
